src/sqlfluff/core/rules/__init__.py

```diff
@@ -25,8 +21,25 @@
         __all__.append(rule.__name__)
 
 
+def _load_standard_rules():
+    """Initialise the standard ruleset.
+
+    We do this on each call so that dynamic rules changes
+    are possible.
+    """
+    std_rule_set = RuleSet(name="standard", config_info=STANDARD_CONFIG_INFO_DICT)
+
+    # Iterate through the rules list and register each rule with the standard set.
+    for plugin_rules in get_plugin_manager().hook.get_rules():
+        for rule in plugin_rules:
+            std_rule_set.register(rule)
+
+    return std_rule_set
+
+
 def get_ruleset(name: str = "standard") -> RuleSet:
     """Get a ruleset by name."""
-    lookup = {std_rule_set.name: std_rule_set}
+    std_rules = _load_standard_rules()
```

Comment:
Will rule plug-ins be discovered and loaded in each child process or thread? (Context: I want to be know if custom rules and `--processes` work correctly together.)

Reply:
right now they won't - see comments above. I think that would probably need to be addressed as part of a seperate PR.